qscripts-cli/q_plot.py

```diff
@@ -399,15 +403,28 @@ def on_resize(self,event):
             from matplotlib.figure import Figure
             import matplotlib.patches as mpatches
             from mpl_toolkits.mplot3d import Axes3D
+        
         except ImportError:
-            print("""Matplotlib is required for this script to work. Try something like:
-            (ubuntu)    $ sudo apt-get install python-matplotlib
-            (mac)       $ sudo brew install matplotlib
-            (mac)       $ sudo port install py27-matplotlib
-            (anything)  $ sudo pip install matplotlib
-            (anything)  $ conda install -c conda-forge matplotlib=2.0.0
-            or if you are working on a cluster, try loading a different python module...""")
-            sys.exit(1)
+            print("Failed to load matplotlib 2.*, trying 3.*")
+            try:
+                import matplotlib
```

Comment:
how does this code detect the other version of matplotlib? the code itself is identical?